ascon/src/util.rs

```diff
@@ -1,17 +1,16 @@
-use byteorder::{BigEndian, ByteOrder};
-
-pub type Endian = BigEndian;
+use core::convert::TryInto;
 
 #[inline]
+#[allow(clippy::unwrap_used)]
 pub fn u8_to_u64(input: &[u8], output: &mut [u64]) {
     for (i, b) in output.iter_mut().enumerate() {
-        *b = Endian::read_u64(&input[(i * 8)..((i + 1) * 8)]);
+        *b = u64::from_be_bytes(input[(i * 8)..((i + 1) * 8)].try_into().unwrap());
```

Comment:
FYI you can sue a more idiomatic version:
```rust
pub fn u8_to_u64(input: &[u8], output: &mut [u64]) {
    assert_eq!(input.len(), 8 * output.len());
    input
        .chunks_exact(8)
        .map(|c| c.try_into().unwrap())
        .map(u64::from_be_bytes)
        .zip(output.iter_mut())
        .for_each(|(inp, out)| *out = inp)
}
```
And in future [`array_chunks`](https://doc.rust-lang.org/std/primitive.slice.html#method.array_chunks) can make it even nicer.